neo/Implementations/Wallets/peewee/UserWallet.py

```diff
@@ -87,21 +84,14 @@ def Create(path, password):
         return wallet
 
     def CreateKey(self, prikey=None):
-        if prikey:
-            private_key = prikey
-        else:
-            private_key = bytes(Random.get_random_bytes(32))
-
-        account = WalletKeyPair(priv_key=private_key)
+        account = super(UserWallet, self).CreateKey(private_key=prikey)
```

Comment:
I'm not 100% sure about this. What if the `Wallet.CreateKey` method changes to do something more? You could just simplify the block on top with `private_key = prikey or bytes(Random.get_random_bytes(32))`

Reply:
my changes actually make it match the original `neo-project` more closely
See: https://github.com/neo-project/neo/blob/1bbebb1db8fa7e5a72f372e933f65f4815a1911b/neo/Implementations/Wallets/EntityFramework/UserWallet.cs#L102

Reply:
Thanks, good to have this context!